### ISSUE TITLE
Engine Brake sounds not working

### ISSUE THREAD

**Issue by MypzMod** [dariowouters/ts-fmod-plugin#31]
There's an issue with the events related to triggering the engine brake sounds, from what I tested, engine brake sounds are only being played when the retarded is also activated.

**Comment by ROSZCO** [comment]
I found for my self to that it's not even recognising the g29 foot pedals in the options to  assign a button in ATS settings so when you lift off ya throttle the Jake works. So we got some issues for sure

**Comment by MypzMod** [comment]
> I found for my self to that it's not even recognising the g29 foot pedals in the options to assign a button in ATS settings so when you lift off ya throttle the Jake works. So we got some issues for sure

In that case, I believe you have the auto engine brake option on, either from the key binding or from the gameplay options. When auto jake is on the manual switch will not work.

**Comment by ROSZCO** [comment]
> > I found for my self to that it's not even recognising the g29 foot pedals in the options to assign a button in ATS settings so when you lift off ya throttle the Jake works. So we got some issues for sure
> 
> In that case, I believe you have the auto engine brake option on, either from the key binding or from the gameplay options. When auto jake is on the manual switch will not work.

Nah its all unticked but thanks it was good to check that, but even if i try to add my pedals to any option they are not recognized except for using them as what they are intended. so something is up the crappo but i think its steam aka ATS and them messing with all the controller settings

**Comment by MypzMod** [comment]
> Nah its all unticked but thanks it was good to check that, but even if i try to add my pedals to any option they are not recognized except for using them as what they are intended. so something is up the crappo but i think its steam aka ATS and them messing with all the controller settings

I had a friend with a similar issue. Since this new update changed the mapping for controllers I see some old configurations not working properly with 1.51... I suggest you forward your issue to the SCS forum, since your problem is unrelated to the FMOD plugin I highly doubt that you will find an answer here.

**Comment by ROSZCO** [comment]
> > Nah its all unticked but thanks it was good to check that, but even if i try to add my pedals to any option they are not recognized except for using them as what they are intended. so something is up the crappo but i think its steam aka ATS and them messing with all the controller settings
> 
> I had a friend with a similar issue. Since this new update changed the mapping for controllers I see some old configurations not working properly with 1.51... I suggest you forward your issue to the SCS forum, since your problem is unrelated to the FMOD plugin I highly doubt that you will find an answer here.

yeah i realize that i just posted so others might see why its not working as intended. Thanks for ya advice
